install/framework/nunit-framework.wxs

```diff
@@ -58,7 +58,7 @@
     
 	<Fragment>
 		<Directory Id="TARGETDIR" Name="SourceDir">
-			<Directory Id="ProgramFilesFolder">
+			<Directory Id="ProgramFiles64Folder">
```

Comment:
Yup! But what happens on a non-64-bit system? IOW, do we have to make the
distinction or does WiX do it for us.

On Wed, Jan 21, 2015 at 2:02 PM, Rob Prouse <notifications@github.com>
wrote:

> In install/framework/nunit-framework.wxs
> <https://github.com/nunit/nunit/pull/466#discussion_r23338456>:
>
> > @@ -58,7 +58,7 @@
> >
> >  	<Fragment>
> >  		<Directory Id="TARGETDIR" Name="SourceDir">
> > -			<Directory Id="ProgramFilesFolder">
> > +			<Directory Id="ProgramFiles64Folder">
>
> ProgramFiles64Folder goes to Program Files, ProgramFilesFolder goes to Program
> Files (x86).
>
> It confused me, same? [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/466/files#r23338456>.
>